migrations/env.py

```diff
@@ -0,0 +1,86 @@
+import yaml
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from logging.config import fileConfig
+
+# import os
+# import sys
+#
+# sys.path.append(os.getcwd())
+#
+# from ras_party.models.models import Base
+
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# override sqlalchemy.url
+config_path = 'config/config.yaml'
+with open(config_path, 'r') as f:
+    config_data = yaml.load(f)
```

Comment:
this should be `yaml.safe_load(f)`

Using `yaml.load` potentially allows arbirtary code execution 
https://security.openstack.org/guidelines/dg_avoid-dangerous-input-parsing-libraries.html

Reply:
We won't be using yaml. Once @jcox-dev's config refactor goes in, so i'll probably just leave this in for now.